Assets/Scripts/WinOrLoseScript.cs

```diff
@@ -54,88 +164,266 @@ void Start()
+            {//if not first player (blocks error)
+                if(IsTied(players[i], players[i - 1]))
+                {//if player is tied with the one before it
+                    ReplacePodium(i);
+
+                    if (IsTied(players[i], winners[0]))
+                    {//if player is tied with the winner
+                        winners.Add(players[i]);
+                    }
+                }
+            }
+        }
 
-        podiums[players.Length-1].gameObject.SetActive(false);
+        PopulateLosers();
     }
+    #endregion
 
-    Player[] Order(Player[] scores)
+    private void PopulateLosers()
     {
-        for(int current = 0; current < scores.Length; current++)
+        foreach(Player player in players)
         {
-            for(int after = current + 1; after < scores.Length; after++)
+            if (!winners.Contains(player))
+            {//if not a winner
+                losers.Add(player);
+            }
+        }
+    }
+
+    private List<int> GenerateBigestLosersList()
+    {
+        if(losers.Count == 0)
+        {
+            return new List<int>(new int[] { 0, 1, 2, 3 });
+        }
+
+        int index = losers.Count - 1; //starts on player in last
+
+        //Generate list of last place players
+        List<int> podiumsToDestroy = new List<int>();
+        podiumsToDestroy.Add(index); //add known biggest loser to list
+
+        while (index - 1 != -1 && IsTied(losers[index], losers[index - 1]))
+        {//while in bounds and has the same score as last place person
+            //add to the destroy list
+            podiumsToDestroy.Add(index - 1);
+
+            //setup for next loop
+            index--;
+        }
+
+        return podiumsToDestroy;
+    }
+
+    void Order()
+    {
+        for(int current = 0; current < players.Length; current++)
+        {
+            for(int after = current + 1; after < players.Length; after++)
             {
-                if(scores[after].score > scores[current].score)
+                if(players[after].score > players[current].score)
                 {
-                    //swap scores
-                    Player temp = scores[current];
-                    scores[current] = scores[after];
-                    scores[after] = temp;
+                    //swap players
+                    Player temp = players[current];
+                    players[current] = players[after];
+                    players[after] = temp;
                 }
             }
         }
+    }
+    #endregion
 
-        return scores;
+    #region Win or Lose
+    void PlayWinOrLose(Player player)
+    {
+        if (winners.Contains(player))
+        {//if local player won
+            Win(player);
+            return;
+        }
+        
+        //otherwise if you didnt win, oyu lost
+        Lose();
     }
-    
-    void Win()
+
+    void Win(Player winner)
     {
-        audioSource.PlayOneShot(winSound);
+        PlaySound(audioVaraibles.winSound);
```

Comment:
we don't have a win/lose sound, these might cause errors temporarily

Reply:
I put the temp sounds in to avoid the error